docs/reference/cat/dataframeanalytics.asciidoc

```diff
@@ -5,6 +5,13 @@
 <titleabbrev>cat {dfanalytics}</titleabbrev>
 ++++
 
+[IMPORTANT]
+====
+cat APIs are only intended for human consumption using the command line or {kib} 
+console. They are _not_ intended for use by applications. For application 
+consumption, use the <<get-dfanalytics-stats>>.
```

Comment:
Some of these link titles contain "API" and some are missing them and therefore have an incomplete phrase (e.g. https://elasticsearch_83382.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/cat-dfanalytics.html). It might be simplest to just use an explicit link title for all of them. For example:
```suggestion
consumption, use the <<get-dfanalytics-stats,get data frame analytics jobs stats API>>.
```